src/controllers/MetaDataController.ts

```diff
@@ -273,7 +276,10 @@ export class MetaDataController {
       const socialPictureValue = resolution.resolution['social.picture.value'];
       const pfpImageFromCDN =
         socialPictureValue &&
-        (await getNftPfpImageFromCDN(socialPictureValue, withOverlay));
+        (await getNftPfpImageFromCDN(
```

Comment:
Consider changing API to, to move such ternary operator into the `getNftPfpImageFromCDN` function:
```suggestion
        (await getNftPfpImageFromCDN(
          {
            socialPictureValue,
            withOverlay,
            domainName: domain.name,
          }
        ))
```